### PR TITLE
Tweaks to pipe (metal+copper), crowbar (makeshift+normal), Halligan Tool and New York Hook

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
Balance "Changed volume and weight to pipes, crowbars, halligan tool and new your hook and added max lengh to pipes"

#### Purpose of change
Pipes were still 10 cm. I wanted to make them longer. I took the makeshift crowbar as its example. After that, I used the power of math (cylinder) and some references to change volume to said objects. I also tweaked the weight a bit as well.

#### Additional context
https://www.amazon.de/Connex-COX664600-Nageleisen-sechskant-L%C3%A4nge/dp/B001BAWDP8/ref=sxin_10?ascsubtag=amzn1.osa.a97a059a-883d-43c6-80ed-0850dfce3426.A1PA6795UKMFR9.de_DE <- for the real crowbar
https://www.hornbach.de/shop/Edelstahlrohr-Viega-Sanpress-1-4401-28x1-2-mm-2-m/6796449/artikel.html <- for the metal pipe
https://www.rescue-tec.de/Halligan-Tool-Hebelklaue.html <- Halligan Tool. I added the top and lowest part accordingly in the volume calculation.
https://www.heimwerker.de/stahlgewicht-berechnen/ <- for the new york hook, which is made out of steel alone, I took another approach. I looked up the weight of it (which is mostly correct ingame) and I checked the kg/m³ of steel which is 8. And I then calculated the volume based on that (round about).

### CODE REVIEW

**Comment by BrettDong** [comment]
May need to update unit test.

**Comment by Lamandus** [comment]
> 
> 
> May need to update unit test.

oh? how so?

File: data/json/items/resources/metal.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "material": [ "steel" ],
     "qualities": [ [ "HAMMER", 1 ] ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "1 L",
+    "volume": "370 ml",
```
Comment (RadHazard) [inline data/json/items/resources/metal.json:15]: I think you missed adding longest side to the regular pipe

Comment (Lamandus) [reply data/json/items/resources/metal.json:15]: I added it, but I did some undo and it vanished. Thanks.

**Comment by anothersimulacrum** [comment]
The crowbar is too heavy to fit into tool belts now, so the pocket wieght max should be increased.
Only the `test_tool_belt` needs to adjustment, but please make it for the normal toolbelt as well.

**Comment by Lamandus** [comment]
> 
> 
> The crowbar is too heavy to fit into tool belts now, so the pocket wieght max should be increased.
> Only the `test_tool_belt` needs to adjustment, but please make it for the normal toolbelt as well.

I checked the toolbelt in the json. The crowbars didn't fit already in the past, because the lengh of both were over the allowed 50 cm. The weight is not a problem (max 1500 g, both are below that).

File: data/json/items/resources/metal.json [hunk 2]
```diff
@@ -6,13 +6,14 @@
     "description": "A steel pipe, makes a good melee weapon.  Useful in a few crafting recipes.",
```
Comment (actual-nh) [inline data/json/items/resources/metal.json:6]:
```suggestion
    "description": "A steel pipe; makes a good melee weapon.  Useful in a few crafting recipes.",
```

File: data/json/items/resources/metal.json [hunk 3]
```diff
@@ -59,13 +60,14 @@
     "name": { "str": "copper tubing" },
     "description": "A copper tube, too thin to be much use as a melee weapon, but will do if nothing else is available.  Useful in a few crafting recipes.",
```
Comment (actual-nh) [inline data/json/items/resources/metal.json:61]:
```suggestion
    "description": "A copper tube; too thin to be much use as a melee weapon, but will do if nothing else is available.  Useful in a few crafting recipes.",
```

File: data/json/items/tool/entry_tools.json [hunk 4]
```diff
@@ -4,9 +4,9 @@
     "type": "TOOL",
     "name": { "str": "crowbar" },
     "description": "This is a hefty prying tool.  Use it to open locked doors without destroying them or to lift manhole covers.  You could also wield it to bash some heads in.",
```
Comment (actual-nh) [inline data/json/items/tool/entry_tools.json:6]:
```suggestion
    "description": "This is a hefty prying tool.  Use it to open locked doors without destroying them, or to lift manhole covers.  You could also wield it to bash some heads in.",
```

File: data/json/items/tool/entry_tools.json [hunk 5]
```diff
@@ -62,8 +62,8 @@
     "type": "TOOL",
     "name": { "str": "makeshift crowbar" },
     "description": "This is a pipe whose ends have been bent and hammered flat to resemble a crowbar.  Use it to open locked crates without destroying them, or to lift manhole covers.  You could also wield it to fight with, in a pinch.",
```
Comment (actual-nh) [inline data/json/items/tool/entry_tools.json:64]:
```suggestion
    "description": "This is a pipe with ends that have been bent and hammered flat so it resembles a crowbar.  Use it to open locked crates without destroying them, or to lift manhole covers.  You could also wield it to fight with, in a pinch.",
```

**Review by actual-nh — COMMENTED** [review]
Copyediting.

File: data/json/items/resources/metal.json (hunk 2 above)
Comment (Lamandus) [reply data/json/items/resources/metal.json:6]: not sure why we need to change things that weren't changed from me in the first place. But ok, I am no native English speaker, I trust you on that one.

**Comment by Lamandus** [comment]
I also took my time to update the wrong volume of the halligan bar and NY Hook.

**Comment by anothersimulacrum** [comment]
Consider dropping the string changes? Otherwise this can't make it in before 0.F.

**Comment by Lamandus** [comment]
> 
> 
> Consider dropping the string changes? Otherwise this can't make it in before 0.F.

which string changes?

**Comment by Lamandus** [comment]
@anothersimulacrum you can remove the string freeze label now, so I may get the chance to put it in the game before F.

**Comment by actual-nh** [comment]
The halligan bar is among those used for the dps test - seems to need updating for new stats. And there's still an error about not being able to fit into a tool belt, it looks like.

**Comment by Lamandus** [comment]
> 
> 
> The halligan bar is among those used for the dps test - seems to need updating for new stats. And there's still an error about not being able to fit into a tool belt, it looks like. (Perhaps I should have edited the test tool belt in #47706 instead of the item info test?)

Would be an idea.
But about the DPS test there is something I don't get.
`calc_expected_dps( test_guy, "warhammer", 28.0 );`
I spawned a warhammer with a 10 10 10 10 character and 4 in bashing and meele (as the test_guy should have). I get: 
grafik

but in the code the expected DPS is only 28? Or am I reading it wrong?

**Comment by Lamandus** [comment]
> 
> 
> The halligan bar is among those used for the dps test - seems to need updating for new stats. And there's still an error about not being able to fit into a tool belt, it looks like. (Perhaps I should have edited the test tool belt in #47706 instead of the item info test?)

right, I did the changes accordingly. About the tool belt, I leave it to your PR, ok?

**Comment by actual-nh** [comment]
It is incredibly irritating how little information Github gives one regarding merge conflicts.

**Comment by Lamandus** [comment]
I put back those string changes again @actual-nh , since I got in freeze again.

**Comment by Lamandus** [comment]
I will split those for better workflow.

**Pull request closed without merging**